patches/PR3/config.guess.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/build-aux/config.guess b/build-aux/config.guess
+index 7f76b62..9eab862 100755
+--- a/build-aux/config.guess
++++ b/build-aux/config.guess
+@@ -109,7 +109,7 @@ GUESS=
+ 
+ tmp=
+ # shellcheck disable=SC2172
+-trap 'test -z "$tmp" || rm -fr "$tmp"' 0 1 2 13 15
++trap 'test -z "$tmp" || rm -fr "$tmp"' EXIT HUP INT PIPE TERM
```

Comment:
I wonder if we can also fix the source of this, which I think comes from autoconf

Reply:
Interesting point.
I didn't realise that was a generated file as it is already in that state in the tar file.
Presumably it would need to be fixed in autoconf if we were to try to switch to the git source.

Reply:
Looking into this it has already been fixed in autoconfport.
See [config.guess.patch](https://github.com/ZOSOpenTools/autoconfport/blob/main/patches/PR1/config.guess.patch) which was added by https://github.com/ZOSOpenTools/autoconfport/pull/13